fatukunda/investiment-club-api
# # View user profile - Enables users to view profile

#### What does this PR do?
 - Enables the user to view their profile
#### Description of Task to be completed?
 - Create an auth middleware.
 - Create a view user profile route.
 - Create a view user profile controller.
 - Write unit tests for the user profile feature.
#### How should this be manually tested?
 - Navigate to `localhost:3000/api/v1/users/me`
 - Provide the `Authorization header` and a `Bearer token`
 - Hit send.
 - If all goes well, you should be able to see the user data. 
#### What are the relevant pivotal tracker stories?
[#164983331](https://www.pivotaltracker.com/story/show/164983331)
#### Any background context you want to add?
 - A user should be logged in to view their profile. Logging in will provide you with the Bearer token to use in the Authorization header.
<img width="1142" alt="Screen Shot 2019-12-18 at 12 31 46 PM" src="https://user-images.githubusercontent.com/4363632/71073922-63b57680-2192-11ea-8dde-3dba054a1b88.png">

#### Screenshots
